earthpy/tests/test_spatial.py

```diff
@@ -290,4 +303,51 @@ def test_crop_image_fails_empty_list(basic_image_tif, basic_geometry):
         es.crop_image(list(), basic_geometry)
     with rio.open(basic_image_tif) as src:
         with pytest.raises(ValueError):
+
             es.crop_image(src, list())
+
+
+def test_hillshade_shape(hillshade_arr):
+    """If provided with input array not of shape (rows, columns),
```

Comment:
Again this message is confusing to me as rows, columns is not a shape. Do you mean a 2-dimensional array?

Reply:
@lwasser This is the terminology that was already present in the function (e.g. "of shape (rows, columns"), so I didn't change it. I will change it now to be more clear that the function requires a 2-dimensional array.